android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.at_dude">
-    <application android:label="@dude" android:name="${applicationName}" android:icon="@mipmap/launcher_icon">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.atsign.atdude">
+    <application android:label="AtDude" android:name="${applicationName}" android:icon="@mipmap/launcher_icon">
```

Comment:
I thought the naming convention was "atDude" with a lowercase "a" @jenmonroe please confirm.